test/scripts/e2e-kind.sh

```diff
@@ -10,8 +10,43 @@ export KUBECONFIG=${HOME}/admin.conf
 export MASTER_NAME=${KIND_CLUSTER_NAME}-control-plane
 export NODE_NAMES=${MASTER_NAME}
 
-#REMOVE skip for "ingress access from updated pod" when k8s is updated to 1.18
-SKIPPED_TESTS='Networking\sIPerf\sIPv[46]|\[Feature:PerformanceDNS\]|\[Feature:IPv6DualStackAlphaFeature\]|\[Feature:NoSNAT\]|Services.+(ESIPP|cleanup\sfinalizer|session\saffinity)|\[Feature:Networking-IPv6\]|\[Feature:Federation\]|configMap\snameserver|ClusterDns\s\[Feature:Example\]|kube-proxy|should\sset\sTCP\sCLOSE_WAIT\stimeout|EndpointSlices|named\sport.+\[Feature:NetworkPolicy\]|should\sallow\singress\saccess\sfrom\supdated\spod.+\[Feature:NetworkPolicy\]'
+SKIPPED_TESTS="
+# PERFORMANCE TESTS: NOT WANTED FOR CI
+Networking IPerf IPv[46]
+\[Feature:PerformanceDNS\]
+
+# FEATURES NOT AVAILABLE IN OUR CI ENVIRONMENT
+\[Feature:Networking-IPv6\]
+\[Feature:Federation\]
+
+# TESTS THAT ARE IRRELEVANT FOR OVN-KUBERNETES
+kube-proxy
+
+# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/1142
+\[Feature:IPv6DualStackAlphaFeature\]
+
+# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/819
+# Services.+session affinity
+
+# TO BE IMPLEMENTED: https://github.com/ovn-org/ovn-kubernetes/issues/1116
+EndpointSlices
+
+# REMOVE when k8s is updated to 1.18
+should allow ingress access from updated pod.+\[Feature:NetworkPolicy\]
+
+# NOT IMPLEMENTED; SEE DISCUSSION IN https://github.com/ovn-org/ovn-kubernetes/pull/1225
+named port.+\[Feature:NetworkPolicy\]
+
+# ???
+\[Feature:NoSNAT\]
+Services.+(ESIPP|cleanup finalizer|session affinity)
+configMap nameserver
```

Comment:
no idea on this one or line 44
@as-com do you know why these tests were skipped?

Reply:
[sig-network] NoSNAT [Feature:NoSNAT] [Slow] [It] Should be able to send traffic between Pods without SNAT 

[sig-network] Services [It] should have session affinity work for LoadBalancer service with ESIPP off [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should be able to switch session affinity for LoadBalancer service with ESIPP on [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should have session affinity work for LoadBalancer service with ESIPP on [Slow] [DisabledForLargeClusters] 
[sig-network] Services [It] should be able to switch session affinity for LoadBalancer service with ESIPP off [Slow] [DisabledForLargeClusters] 

**[sig-network] DNS configMap nameserver [Feature:Networking-IPv6] [LinuxOnly]** 
Change stubDomain [It] should be able to change stubDomain configuration [Slow][Serial] 
Forward PTR lookup [It] should forward PTR records lookup to upstream nameserver [Slow][Serial] 
Forward external name lookup [It] should forward externalname lookup to upstream nameserver [Slow][Serial] 
**[sig-network] DNS configMap nameserver [IPv4]** 
Forward external name lookup [It] should forward externalname lookup to upstream nameserver [Slow][Serial] 
Forward PTR lookup [It] should forward PTR records lookup to upstream nameserver [Slow][Serial] 

these are from [my notes](https://docs.google.com/document/d/1BOKGgsp7rwlP-Cn56DPcgbpkd1HADb6zlUA0ZrBa0OQ/edit#) on which tests are failing - does this help?

Reply:
thanks @as-com it does help identify the individual test cases. For the DNS ones I have never looked at those, so not sure if those should pass or fail. Need to check.